app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
   <div>
-    <p><%= banner.heading %><br>
-    <%= banner.short_description %></p>
+    <h1><%= banner.heading %></h1>
```

Comment:
Ideally there should only be one h1 per page, and that should reflect what the page is about.
I recently tried to find proper evidence of the why and how but also had trouble finding anything substantial. (I'm currently investigating something similar.) I know some screen reader users would jump to the first h1 and not assume there is another one.

Semantically I would say the emergency banner heading doesn't make sense to be an h1, especially since it will be the same across multiple pages. What was the reason for it? I'd suggest using an h2.
(Although it's also controversial to use any heading before the h1. But I think giving structure is more important than being strict about the hierarchy.)

Sorry, that's not a very clear answer. The main problem is that HTML doesn't have a concept of structure and heading hierarchy outside of the main content. The W3C tried solving that with the infamous document outline but failed. So, although there might be a better answer, I don't think there will be a 100% answer any time soon.